MLIR.cmake

```diff
@@ -1,30 +1,32 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Path to LLVM source folder.
-if(DEFINED ENV{LLVM_PROJ_SRC})
+if(DEFINED LLVM_PROJ_SRC)
+elseif(DEFINED ENV{LLVM_PROJ_SRC})
   set(LLVM_PROJ_SRC $ENV{LLVM_PROJ_SRC})
-  if(EXISTS ${LLVM_PROJ_SRC})
-    message(STATUS "LLVM_PROJ_SRC           : " ${LLVM_PROJ_SRC})
-  else()
-    message(FATAL_ERROR "The path specified by LLVM_PROJ_SRC does not exist: "
-            ${LLVM_PROJ_SRC})
-  endif()
 else()
   message(FATAL_ERROR "env variable LLVM_PROJ_SRC not set")
```

Comment:
This message (below as well) is not longer accurate since now the variable can be specified as an environment variable through a cmake file or as part of the cmake command with -DLLVM_PROJ_SRC=...

Reply:
good point...